lib/search_vector.rb

```diff
@@ -5,10 +5,15 @@ module SearchVector
   AcceptedTables = ['supporters', 'payments']
 
   def self.query(query_string, expr=nil)
-    (expr || Qexpr.new).where(
+    query = if query_string.is_a?(Integer) || query_string.is_int?
+      "(supporters.fts @@ websearch_to_tsquery('english', $search::varchar(255))
```

Comment:
When you didn't case to a varchar, did it cause problems here?

Reply:
Yes, if I pass an integer I get an error:

```
ActiveRecord::StatementInvalid:
       PG::UndefinedFunction: ERROR:  function websearch_to_tsquery(unknown, integer) does not exist
```

Although when it comes from the interface, the query already comes as a string, I think it won't hurt to avoid problems.

Reply:
Hmmm, that makes sense then.